pkgs/top-level/python-packages.nix

```diff
@@ -37,6 +37,20 @@ let
 
   graphiteVersion = "0.9.15";
 
+  fetchPypiMirror = ({ name ? null , pname ? null, version ? null, kind, ext, sha256}:
+    let
+      _name = if name == null then "${pname}-${version}" else name;
+      _pdrv = builtins.parseDrvName _name;
+    in
+      pkgs.fetchurl {
+        inherit sha256;
+        url = "mirror://pypiio/${kind}/${builtins.substring 0 1 _name}/${_pdrv.name}/${_name}${ext}";
```

Comment:
I know it should work with normalized names, but my experience tells me different :)